### DISCUSSION TITLE
如何某个taget不继承全局根作用域影响

### DISCUSSION THREAD

**Discussion by PeakRacing** [xmake-io/xmake#4323] [Q&A] [answered]
如何某个taget不继承全局根作用域影响
比如全局根作用域 设置了很多通用的c ld flags和定义 ，其中有一个taget不想继承，剩下的那些都正常继承，如何指定某个taget不继承？
@waruqi

**Comment by waruqi** [comment, marked as answer]
拆分到多个子 xmake.lua ，在 子 xmake.lua 的根域设置 仅用于这些 target 的全局配置，其他平级的 xmake.lua 里面的 target 就不会被继承到。。否则你只能不放全局域，自己走 function wrap 挨个 target 设置
